lib/grandpa/types.go

```diff
@@ -34,43 +32,23 @@ type (
 	SignedVote = types.GrandpaSignedVote
 )
 
-type subround byte
+// Subround subrounds in a grandpa round
+type Subround byte
 
+//nolint
 var (
-	prevote         subround
-	precommit       subround = 1
-	primaryProposal subround = 2
+	Prevote         Subround
+	Precommit       Subround = 1
+	PrimaryProposal Subround = 2
```

Comment:
why are these public now?

Reply:
So scale can access them when decoding/encoding. I believe this is necessary but I can change if I am wrong

Reply:
The values don't need to be public, only the attribute names if they are being used as an attribute in a struct.